build-util/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "build-util"
```

Comment:
Could i ask why rename it? It's not meant for publishing since it works only inside the repo, which has it's code anyway. I was doing something like [cargo-xtask](https://github.com/matklad/cargo-xtask) approach, or more precisely like [rust-analyzer does](https://github.com/rust-lang/rust-analyzer/blob/609bdbc4ddcd40d480730876595c0b7f4a58ffcc/xtask/Cargo.toml). On that note, ra seems to set `publish = false`.

Reply:
It's `vitasdk-sys` dependency, which we have to publish, you're right.